Tests/FuzzilliTests/MutationsTest.swift

```diff
@@ -52,12 +52,17 @@ class MutationsTests: XCTestCase {
         }
         b.endIf()
         let program = b.finalize()
-        program.runtimeTypes = VariableMap<[Int: Type]>([[0: .number], [1: .boolean], [3: .float]])
+        program.runtimeTypes = ProgramTypes(
+            from: VariableMap<Type>([.number, .boolean, .float]),
```

Comment:
Nit: here and elsewhere, I think you can leave off the `<Type>`, the compiler should be able to infer that